firebase-app-distribution/src/test/java/com/google/firebase/app/distribution/FirebaseAppDistributionTest.java

```diff
@@ -220,7 +223,8 @@ public void updateToNewRelease_whenNewAabReleaseAvailable_showsUpdateDialog() {
     AppDistributionReleaseInternal newRelease = TEST_RELEASE_NEWER_AAB_INTERNAL.build();
     when(mockCheckForNewReleaseClient.checkForNewRelease()).thenReturn(Tasks.forResult(newRelease));
     firebaseAppDistribution.setCachedNewRelease(newRelease);
-    when(mockUpdateAppClient.updateApp(newRelease, true)).thenReturn(new UpdateTaskImpl());
+    //  when(firebaseAppDistribution.updateApp(true)).thenReturn(new UpdateTaskImpl());
+    doReturn(new UpdateTaskImpl()).when(firebaseAppDistribution).updateApp(true);
```

Comment:
Instead of mocking the updateApp method in firebaseAppDistribution, could we set the return value of the mock updateAAB client? Mocking the updateApp method  means that we don't test the code in the class under test